libs/application/templates/parental-leave/src/fields/components/Table/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { FC, PropsWithChildren, ReactElement, useState } from 'react'
```

Comment:
this table component looks awesome. It will most definitely mov into the ui kit at some time in the future